tests/virtualization/yast_virtualization.pm

```diff
@@ -50,7 +51,17 @@ sub run {
     x11_start_program('xterm');
     wait_screen_change { send_key 'alt-f10' };
     become_root;
+    if (is_leap('15.0+')) {
+        systemctl 'is-active virtqemud.socket', timeout => 60;
```

Comment:
@alice-suse 

these 2 are active, so definitely the "modular daemons are the default" because If either of the above is active your system is using the modular daemons.


clearly, it is verified here: https://openqa.opensuse.org/tests/3785356#step/yast_virtualization/26

I will commit one more, but we need libvirt to be installed, because restarting modular libvirt is not enough for virt-manager and restarting is also failing sometimes. HERE: https://openqa.opensuse.org/tests/3785057#step/yast_virtualization/40

Reply:
I am afraid it is hard to tell if this PR fixes the root cause, given above failure. To me, we still need to check out what kind of libvirt daemon is the default for this leap product, make the default one installed, started/enabled as it defines, then fix whatever part of automation that does not fit and file product bug if the product is not performing as expected.